build

```diff
@@ -33,16 +33,12 @@ etcd_build() {
 	out="bin"
 	if [ -n "${BINDIR}" ]; then out="${BINDIR}"; fi
 	toggle_failpoints
-	# Static compilation is useful when etcd is run in a container
-	CGO_ENABLED=0 go build $GO_BUILD_FLAGS -installsuffix cgo -ldflags "-s -X ${REPO_PATH}/cmd/vendor/${REPO_PATH}/version.GitSHA=${GIT_SHA}" -o ${out}/etcd ${REPO_PATH}/cmd
-	CGO_ENABLED=0 go build $GO_BUILD_FLAGS -installsuffix cgo -ldflags "-s" -o ${out}/etcdctl ${REPO_PATH}/cmd/etcdctl
+	go build $GO_BUILD_FLAGS -installsuffix cgo -ldflags "-s -X ${REPO_PATH}/vendor/${REPO_PATH}/version.GitSHA=${GIT_SHA}" -o ${out}/etcd ${REPO_PATH}
+	CGO_ENABLED=0 go build $GO_BUILD_FLAGS -installsuffix cgo -ldflags "-s" -o ${out}/etcdctl ${REPO_PATH}/etcdctl
 }
 
 etcd_setup_gopath() {
-	export GOPATH=${CDIR}/gopath
-	rm -f $GOPATH/src
-	mkdir -p $GOPATH
-	ln -s ${CDIR}/cmd/vendor $GOPATH/src
+	cp -rf ${CDIR}/vendor-release ${CDIR}/vendor
```

Comment:
I think it can be either symlink as we do now, or `vendor-release` directory (then no need for symlinks). Think this is easier when we deal with Windows?